### ISSUE TITLE
Handlebars | 1:1 ?

### ISSUE THREAD

**Issue by jbuzzanco** [ga-wdi-boston/full-stack-project#540]
Believe that we are supposed to schedule one on one's thru this issue tracker, I don't know if it would be like at the regular 4pm time or something. Was wondering if it would be possible for someone  to talk about handlebars with me today if there's any open time left.  😃 

I have used it somewhat in my project so far, and would like to understand it more for how I can implement it for specific data attributes, if that is worded correctly or makes sense. I've been following along with the handlebars repo with it in my project. 

So basically was wondering if someone can help with handlebars if possible like as a regular 1:1 whether or not if it's working directly with the project, so that I can know how to better implement it.

**Comment by raq929** [comment]
What have you tried so far?

**Comment by raq929** [comment]
(Yes, we can do a 1:1, but I want to know where you are first.)

**Comment by jbuzzanco** [comment]
Like in code? Ok ya so basically I got it to show all the quotes with handlebars...trying to figure out how to target already created data associated with the user who is logged in to edit.

**Comment by jbuzzanco** [comment]
There is a fair chance I am over-complicating it

**Comment by jbuzzanco** [comment]
For instance, here is what I am trying now to show a delete but if user.id equals quote.user.id, however it is breaking the formatting upon page load
handlebars
```
  <div>
    {{#if quote.user.id === user.id}}
      <button data-id="{{quote.id}}" class="delete-button">Delete This Post</button>
    {{else}}
      Unknown quoteText.
    {{/if}}
  </div>
```

**Comment by jbuzzanco** [comment]
It's also a little hard to know correct formatting bc the linter doesn't care about handlebars.

**Comment by raq929** [comment]
What do you mean 'breaking the formatting'?

**Comment by jbuzzanco** [comment]
The styling of the page is gone upon loading that delete part.

**Comment by raq929** [comment]
I'm not sure handlebars supports using === 
I think you'll want an editable field in your data from the server, and you can use that for your if statement. I'll come by. Where are you?

**Comment by jbuzzanco** [comment]
Ok ya I tried just one equal = too. In the dugout/kitchen area.

**Comment by jbuzzanco** [comment]
Added the   
```
def editable
    scope == object.user
  end
```
in quote serializer and :editable so that the user logged in could edit their stuff.
also added token in the gets quote api
